pkg/k8shandler/visualization.go

```diff
@@ -446,5 +446,43 @@ func isKibanaDifferent(current *apps.Deployment, desired *apps.Deployment) (*app
 		different = true
 	}
 
+	if isImageDifference(current, desired) {
+		current = updateCurrentImages(current, desired)
+		logrus.Infof("Kibana image(s) change found, updating %q", current.Name)
+		different = true
+	}
+
 	return current, different
 }
+
+func isImageDifference(current *apps.Deployment, desired *apps.Deployment) bool {
```

Comment:
Isn't this really a util function?  Nothing visualization specific here.

Reply:
Yes... but currently the only `Deployment` is from Visualization